ocean_lib/models/erc721_factory.py

```diff
@@ -263,3 +281,31 @@ def get_token_address(self, tx_id: str):
         )
 
         return registered_event[0].args.newTokenAddress
+
+    def create_nft_erc_tokens_once(
```

Comment:
what is this? Is this a function that creates the nft and erc at the same time, except you are doing 2 transactions? This function already exists if you scroll up, with just one transaction, calling it directly from the contracts.

Reply:
This function calls `create_nft_with_erc` and handles both events in order to retrieve the address for both token. We did the same for NFT creation and for datatoken as well.